src/operation.rs

```diff
@@ -894,7 +894,7 @@ fn process_unary_operation(
                 Term::Num(n) => Ok(Term::Str(n.to_string())),
                 Term::Str(s) => Ok(Term::Str(s)),
                 Term::Bool(b) => Ok(Term::Str(b.to_string())),
-                Term::Enum(id) => Ok(Term::Str(id.to_string())),
+                Term::Enum(s) => Ok(Term::Str(s.to_string())),
```

Comment:
Same question as before. The thing inside `Term::Enum` is still of type `Ident`, not of type `String`.

Reply:
same as before